src/utils/helpers.js

```diff
@@ -254,9 +254,6 @@ export function tokenInvalid({ storage, client_id, reauth }) {
   if (!credentials.access_token)
     return handleInvalid('Token status: credentials missing access_token')
 
-  if (credentials.client_id !== client_id)
-    return handleInvalid('Token status: client_id mismatch')
```

Comment:
for info, why are we removing this?

Reply:
It's a check for a value that no longer exists. `moltinCredentials` from local storage used to include `client_id` this was removed [here](https://github.com/moltin/js-sdk/blob/f1acaf8c10a74944c87935bbe1e485f64c4717c9/src/factories/request.js#L125). `client_id` can be retrieved from the sdk config.

The only way I could think of that the `moltinCredentials` could get out of sync with the current sdk instance config is if the values are mutated and changed e.g. sdk.config.client_id = "123". This doesn't seem like a pattern we would want to support.

There is an argument to be made that while we don't encorage people to mutate the sdk instance to change the client id, client secret etc they could be doing it already or do it regardless.

In that case we could protect them from themselves by performing this mismatch check and keeping the `client_id` value as part of the `moltinCredentials` local storage entry.

I'm not too invested either way as `moltinCredentials` is already being used for multiple different data shapes. What would you prefer to do here?

Reply:
im really just interested in how it got in there in the first place... what was the motivation for doing the check 🤔 is there something in CM manually manipulating that localstorage??